Resources/Prototypes/Entities/Mobs/Player/skeleton.yml

```
@@ -51,6 +57,9 @@
     name: ghost-role-information-closet-skeleton-name
     description: ghost-role-information-closet-skeleton-description
     rules: ghost-role-information-freeagent-rules
+    requirements: #SS220 ghostrole-times (20h overall) start
```

```suggestion
    requirements: #SS220 ghostrole-times (20h overall) start
```
Это можно удалить, это чудо точно неоткуда сейчас получить

---

Сейчас да, но если можно будет то время уже как вы выставлено, это плохо?